docs/resources/lb.md

```diff
@@ -63,10 +63,11 @@ resource "scaleway_instance_server" "main" {
     type        = "DEV1-S"
     image       = "debian_bullseye"
     enable_ipv6 = false
+}
 
-    private_network {
```

Comment:
Aldo can you explain on the doc, Why we should be using instance_private_nic instead?